tests/listsAndDropDowns.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { test, expect } from '@playwright/test';
+
+test.describe('Lists and Drop Downs', () => {
+    test.beforeEach(async ({ page }) => {
+        await page.goto('/')
+        await page.getByRole('button', { name: 'Owners' }).click()
+        await page.getByRole('link', { name: 'Search' }).click()
+        await expect(page.getByRole('button', { name: 'Owners' })).toHaveText('Owners');
```

Comment:
When you try to assert the text that is part of the locator, this assertion will never fail as an assertion. The test will fail with a different error, that locator was not found. So the text of the assertion have to be removed from the locator and be something else.